### ISSUE TITLE
Listbox pattern: aria-multiselectable conflicts with the aria spec

### ISSUE THREAD

**Issue by michaelmesser** [w3c/aria-practices#2746]
Should aria-multiselectable be true when using aria-checked in a listbox?

[The listbox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/#wai-ariaroles,states,andproperties) states that:
> If any options are selected, each selected option has either [aria-selected](https://w3c.github.io/aria/#aria-selected) or [aria-checked](https://w3c.github.io/aria/#aria-checked) set to true. No more than one option is selected at a time if the element with role listbox does not have [aria-multiselectable](https://w3c.github.io/aria/#aria-multiselectable) set to true.

However, the spec for [aria-multiselectable](https://www.w3.org/TR/wai-aria-1.2/#aria-multiselectable) states that:
> Indicates that the user may select more than one item from the current selectable descendants.
> Authors SHOULD ensure that selected descendants have the [aria-selected](https://www.w3.org/TR/wai-aria-1.2/#aria-selected) [attribute](https://www.w3.org/TR/wai-aria-1.2/#dfn-attribute) set to true, and selectable descendant have the [aria-selected](https://www.w3.org/TR/wai-aria-1.2/#aria-selected) attribute set to false. Authors SHOULD NOT use the [aria-selected](https://www.w3.org/TR/wai-aria-1.2/#aria-selected) attribute on descendants that are not selectable.

**Comment by accdc** [comment]
I would say no it should not.

The reason being that this pattern within the accessibility API matches the following native control type:
https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.checkedlistbox?view=windowsdesktop-7.0

Checkability isn't the same thing as selectability, where you can arrow around within a single-select listbox and check or uncheck multiple options, which also supports the mixed state for partial checkability as well.

**Comment by michaelmesser** [comment]
> The spec language is inconsistent with itself: there's language about using "checked" in multiselects.

Where do you see this?

> The way the ARIA spec language is written for ARIA multiselectable, it doesn't account for the possibility of using ARIA checked to represent the "selected" state.

My potentially incorrect understanding is that selected and checked are entirely different concepts.

**Comment by mcking65** [comment]
It has long been the case that people have used checked instead of selected in multi-select widgets. We introduced changes to ARIA 1.3 to help resolve some of the confusion and problems related to using checked instead of selected.

So, the ARIA 1.3 [description of the `option` role](https://w3c.github.io/aria/#option) now includes:

>Authors SHOULD indicate selection for option elements using one of the following:
>* An aria-selected value of true  on the selected option within a single-select listbox, and optionally aria-selected values of false  on unselected options.
>* Either aria-selected or aria-checked on all options within a multi-select listbox, with a value of true  on selected options, and a value of false  on unselected options.

We didn't add any language to the description of `aria-multiselectable`. That is clearly an oversight. So, I have raised w3c/aria#1979.

Closing as not related to aria practices.
